feature/experimental/p4rt/internal/p4rtutils/p4rtutils.go

```diff
@@ -194,7 +233,7 @@ func explicitP4RTNodes() map[string]string {
 func P4RTNodesByPort(t *testing.T, dut *ondatra.DUTDevice) map[string]string {
 	t.Helper()
 	if *deviations.ExplicitP4RTNodeComponent {
-		return explicitP4RTNodes()
+		return explicitP4RTNodes(t, dut)
```

Comment:
```suggestion
		if dut.Vendor() == ondatra.CISCO {
			return inferP4RTNodesCisco(t, dut)
		}
		return explicitP4RTNodes(t, dut)
```

But reusing deviations.ExplicitP4RTNodeComponent also means that Cisco will not be able to specify P4RT node names by args. @vosipchu does this warrant a new deviation, or is it okay to repurpose an existing deviation?

Reply:
@liulk 
  Based on our discussion with @vosipchu he wanted a way to get the mapping b/w NPU to I/f without passing it via arg  since  ATE may be connected to different NPU's on different setups